### PR TITLE
test: fix coverage reporter

### PR DESCRIPTION
**Pull request by ronag**

### CODE REVIEW

**Comment by ronag** [comment]
Doesn't feel like it gets reports from all tests... https://app.codecov.io/gh/nodejs/undici/branch/coverage-report

Running `npm run coverage` locally I get more coverage on fetch.

**Comment by ronag** [comment]
How do we make sure we always pick the report from v16?

**Comment by szmarczak** [comment]
codecov merges those reports

**Comment by szmarczak** [comment]
See #985 - maybe this will fix it

**Comment by szmarczak** [comment]
Can you merge #985 into this PR? #985 should solve a conflict between tap and nyc.

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "test:tdd": "tap test/*.js -w",
     "test:typescript": "tsd",
     "coverage": "nyc npm run test",
-    "coverage:ci": "npm run coverage -- --reporter=lcovonly",
+    "coverage:ci": "node scripts/verifyVersion.js 16 && nyc --reporter=lcov npm run test || echo Skipping",
```
Comment (szmarczak) [inline package.json:44]: Can you undo the version check? codecov merges these reports

**Comment by ronag** [comment]
I’ll fix those when I’m back at my computer. What I still don’t like is that we run all the tests twice. Once normally and once for coverage.

should we only run coverage?l (which includes testing)?

**Comment by szmarczak** [comment]
> should we only run coverage?l (which includes testing)?

Yep. :)

**Pull request merged**